api3/notifications/openapi.gen.yaml

```diff
@@ -104,6 +106,19 @@ components:
       - highestSeverity
       title: Short Test Report
       type: object
+    SpecDiffs:
```

Comment:
seems like `SpecDiffs` should be defined in the common swagger since both global and notifications uses it

Reply:
This is how it generated... I only defined the swagger in backend/pkg/modules/internal/differ/restapi/openapi.yaml